framework/wazuh/core/cluster/worker.py

```diff
@@ -698,9 +698,9 @@ async def process_files_from_master(self, name: str, file_received: asyncio.Even
                 logger.debug("Updating local files: End.")
 
             # Send extra valid files to the master.
-            if ko_files['extra_valid']:
+            if 'TYPE' in ko_files and ko_files['TYPE']:
                 logger.debug("Master requires some worker files.")
-                asyncio.create_task(self.sync_extra_valid(ko_files['extra_valid']))
+                asyncio.create_task(self.sync_extra_valid(ko_files['TYPE']))
             else:
                 logger.info(
                     f"Finished in {datetime.utcnow().timestamp() - self.integrity_sync_status['date_start']:.3f}s.")
```

Comment:
Normally this situation will not occur anymore since the extra_valid are disabled, so we can invert the order of this if to avoid an unnecessary check.